ge/views_utils.py

```diff
@@ -496,10 +497,47 @@ def get_librarydata_results(search_type: str, search_term: str) -> list[LibraryD
 
 def sum_col(ws, col, col_top=5):
```

Comment:
Please add type hints for parameters and return type (`None`,in this case).  For `ws`, you can add this import to allow `ws: Worksheet`:
`from openpyxl.worksheet.worksheet import Worksheet`